docs/docs/documentation/tutorials/4-Using_Transformers_in_biome_text.ipynb

```diff
@@ -0,0 +1,1196 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "hHyH5XSkVosk"
+   },
+   "source": [
+    "# Using Transformers in biome.text\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "qgfpoXsuVoso"
+   },
+   "source": [
+    "<a target=\"_blank\" href=\"https://www.recogn.ai/biome-text/documentation/tutorials/4-Using_Transformers_in_biome_text.html\"><img class=\"icon\" src=\"https://www.recogn.ai/biome-text/assets/img/biome-isotype.svg\" width=24 /></a>\n",
+    "[View on recogn.ai](https://www.recogn.ai/biome-text/documentation/tutorials/4-Using_Transformers_in_biome_text.html)\n",
+    "    \n",
+    "<a target=\"_blank\" href=\"https://colab.research.google.com/github/recognai/biome-text/blob/master/docs/docs/documentation/tutorials/4-Using_Transformers_in_biome_text.ipynb\"><img class=\"icon\" src=\"https://www.tensorflow.org/images/colab_logo_32px.png\" width=24 /></a>\n",
+    "[Run in Google Colab](https://colab.research.google.com/github/recognai/biome-text/blob/master/docs/docs/documentation/tutorials/4-Using_Transformers_in_biome_text.ipynb)\n",
+    "        \n",
+    "<a target=\"_blank\" href=\"https://github.com/recognai/biome-text/blob/master/docs/docs/documentation/tutorials/4-Using_Transformers_in_biome_text.ipynb\"><img class=\"icon\" src=\"https://github.githubassets.com/images/modules/logos_page/GitHub-Mark.png\" width=24 /></a>\n",
+    "[View source on GitHub](https://github.com/recognai/biome-text/blob/master/docs/docs/documentation/tutorials/4-Using_Transformers_in_biome_text.ipynb)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "ghPCC7JsCdRq"
+   },
+   "source": [
+    "When running this tutorial in Google Colab, make sure to install *biome.text* first:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "id": "TYh_I8imVosu"
+   },
+   "outputs": [],
+   "source": [
+    "!pip install -U pip\n",
+    "!pip install --use-feature=2020-resolver git+https://github.com/recognai/biome-text.git"
```

Comment:
This install can be normal again I think. We did it to introduce Datasets if I don't remember bad, and that is integrated into biome now.

Reply:
When we added the new `Dataset` we were installing from a branch, here i install from master, so should be fine. But as Dani mentioned in #438 we should link the tutorials to a specific release or at least commit. working on this.

Reply:
Then it is perfect!